### PR TITLE
Money & Ranges

### PR DESCRIPTION
**Pull request by mpdaugherty**
Add native support for money and range messages to protip.

### CODE REVIEW

**Comment by mpdaugherty** [comment]
Look at #14 first; this PR is built on top of that one (though I could separate it out if that's preferable)

File: lib/protip/standard_converter.rb [hunk 1]
```diff
@@ -23,6 +44,13 @@ class << self
       end
     }
 
+    @conversions['protip.messages.Range'] = {
```
Comment (pmen) [inline lib/protip/standard_converter.rb]: this looks like it doesn't belong in this PR, can you confirm if it's needed here?

File: .gitignore [hunk 2]
```diff
@@ -3,3 +3,4 @@
 /gemfiles/*.lock
 /pkg/
 /.bundle/
+/.idea/
```
Comment (pmen) [inline .gitignore]: Would keep this in your local .gitignore

File: definitions/protip/messages/range.proto [hunk 3]
```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
```
Comment (pmen) [inline definitions/protip/messages/range.proto]: I believe you have a separate PR for this

**Comment by mpdaugherty** [comment]
Closed #14 since all the new message types are in this PR.

File: definitions/protip/messages/money.proto [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+    JEP = 165;
+    EEK = 166;
+    MTL = 167;
+    TMM = 168;
+    ZWD = 169;
+    ZWL = 170;
+    ZWN = 171;
+    ZWR = 172;
+  }
+  Currency currency = 2;
```
Comment (tubergen) [inline definitions/protip/messages/money.proto]: probably more DRY if we use a Currency *message* for this, not sure why montag duplicated them on dealer -- perhaps just laziness

Comment (mpdaugherty) [inline definitions/protip/messages/money.proto]: Will do.

**Comment by mpdaugherty** [comment]
Updated. Please take a look when convenient @tubergen @pmen.
